easy_my_coop_taxshelter_report/README.rst

```diff
@@ -20,7 +20,8 @@ Easy My Coop tax shelter report
 |badge1| |badge2| |badge3| 
 
 This module allows you to create a fiscal declaration year and to print
-tax shelter declaration for each cooperator.
+tax shelter declaration for each cooperator. This is a belgian
+particularity.
```

Comment:
When renaming this, let's add "be" to module name. eg `easy_my_coop_be_taxshelter_report`